Lib/pathlib.py

```diff
@@ -418,7 +418,11 @@ def lchmod(self, pathobj, mode):
 
     unlink = os.unlink
 
-    link_to = os.link
+    if hasattr(os, "link"):
+        link_to = os.link
+    else:
+        def link_to(self, target):
```

Comment:
`@staticmethod`

Reply:
Hmm, figured that didn't really matter much since the function is not doing anything with its args anyway. But OK, will fix :)

Reply:
It matters. Without this you will get an error:
```
======================================================================
ERROR: test_link_to_failure (test.test_pathlib.PathTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/serhiy/py/cpython/Lib/test/test_pathlib.py", line 1788, in test_link_to_failure
    p.link_to(q)
  File "/home/serhiy/py/cpython/Lib/pathlib.py", line 1364, in link_to
    self._accessor.link_to(self, target)
TypeError: link_to() takes 2 positional arguments but 3 were given

======================================================================
ERROR: test_link_to_failure (test.test_pathlib.PosixPathTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/serhiy/py/cpython/Lib/test/test_pathlib.py", line 1788, in test_link_to_failure
    p.link_to(q)
  File "/home/serhiy/py/cpython/Lib/pathlib.py", line 1364, in link_to
    self._accessor.link_to(self, target)
TypeError: link_to() takes 2 positional arguments but 3 were given

----------------------------------------------------------------------
```

Reply:
OIC; didn't realise that was how the function was being called. Right, fixed (properly on the second try) :)